src/server/apollo-client.ts

```diff
@@ -36,6 +37,21 @@ const getItemWithExpiry = (key) => {
   return item.value
 }
 
+const addTimezoneOffsetHeader = setContext((_operation, previousContext) => {
+  const { headers, clientTzOffset } = previousContext
+  if (!clientTzOffset) {
+    return previousContext
+  }
+
+  return {
+    ...previousContext,
+    headers: {
+      ...headers,
+      clientTzOffset: new Date().getTimezoneOffset(),
+    },
+  }
+})
```

Comment:
I'd say we should always include it, and rename it to `clientTimezoneOffset`
```suggestion
const addTimezoneOffsetHeader = setContext((_operation, previousContext) => {
  return {
    ...previousContext,
    headers: {
      ...headers,
      clientTimezoneOffset: new Date().getTimezoneOffset(),
    },
  }
})
```